src/pseudopeople/utilities.py

```diff
@@ -134,7 +134,7 @@ def two_d_array_choice(
     """
 
     # Change columns to be integers for datawrangling later
-    options.columns = list(range(len(options.columns)))
+    options = options.rename({col: i for i, col in enumerate(options.columns)}, axis=1)
```

Comment:
Couldn't this be 
```python
options.columns = pd.Index(range(len(options.columns)))
```

Reply:
mypy doesn't like that columns used to contain strs and would now contain ints instead, so rename is a way around that